scout/commands/delete/delete_command.py

```diff
@@ -23,8 +24,18 @@
     "Total variants",
     "Removed variants",
 ]
+VARIANT_CATEGORIES = list(VARIANTS_TARGET_FROM_CATEGORY.keys()) + ["wts_outliers"]
```

Comment:
["wts_outliers"] is added like that, because these variants are on another collection (omics_variant).

Reply:
Right, I must have been blind in that day! Thanks! .)